### PR TITLE
build/barys: Set supervisor tag last

### PR DESCRIPTION
**Pull request by willnewton**
Set the supervisor tag value in local.conf after settig the
default value in order to avoid the value being overwritten.

Signed-off-by: Will Newton <willn@resin.io>

### CODE REVIEW

File: build/barys [hunk 1]
```diff
@@ -396,9 +396,6 @@ else
     # Always activate resinhup bundles in production builds
     sed -i "s/.*RESINHUP ?=.*/RESINHUP ?= \"yes\"/g" conf/local.conf
 fi
-if [ -n "$SUPERVISOR_TAG" ]; then
-    sed -i "s/.*TARGET_TAG ?=.*/TARGET_TAG ?= \"$SUPERVISOR_TAG\"/g" conf/local.conf
-fi
```
Comment (floion) [inline build/barys]: Who can overwrite this?

Comment (willnewton) [inline build/barys]: Further down in this script the value gets overwritten.

Comment (agherzan) [inline build/barys]: I can't find where this variable was overwritten either. Could you point us to a code line?

Comment (willnewton) [inline build/barys]: https://github.com/resin-os/resin-yocto-scripts/blob/master/build/barys#L419
https://github.com/resin-os/resin-yocto-scripts/blob/master/build/barys#L425

The variable in question is TARGET_TAG

**Pull request merged**